### DISCUSSION TITLE
desktop tray icon

### DISCUSSION THREAD

**Discussion by Eui9179** [flet-dev/flet#572] [Q&A]
hello, Thanks for making the `flet`. 

Is there any way to hide the tray icon on the desktop?

**Comment by ndonkoHenri** [comment]
https://flet.dev/docs/controls/page#window_skip_task_bar

**Reply by Eui9179** [reply]
Thank you for your answer.
My question is wrong.
Is there a way to hide it in the taskbar and move it to the tray icon?

**Reply by ndonkoHenri** [reply]
Ah. The tray icon!
I found this thread on stack overflow: https://stackoverflow.com/questions/64305453/flutter-desktop-system-tray-implementation

**Comment by XiangQinxi** [comment]
Maybe this can help you
[pystray](https://pypi.org/project/pystray/)

**Comment by ndonkoHenri** [comment]
https://github.com/ndonkoHenri/Flet-as-System-Tray-Icon
